### PR TITLE
Fixed minor style issues in the docs.

### PR DESCRIPTION
**Pull request by hjafarpour**
Updated the docs based on the new query id format.

### CODE REVIEW

File: docs/quickstart/docker-compose.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ services:
 
   # Runs the Kafka KSQL data generator for topic called "pageviews"
   ksql-datagen-pageviews:
-    image: "confluentinc/ksql-examples:0.2"
+    image: "confluentinc/ksql-examples:0.3-rc1"
```
Comment (apurvam) [inline docs/quickstart/docker-compose.yml]: you probably don't want to push these changes just yet.

Comment (hjafarpour) [reply docs/quickstart/docker-compose.yml]: Yes, removed them.

**Comment by apurvam** [comment]
retest this please

**Comment by hjafarpour** [comment]
retest this please

**Pull request merged**